src/bilder/images/edxapp/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -289,7 +290,7 @@ FEATURES:
     ENABLE_GRADE_DOWNLOADS: true
     ENABLE_INSTRUCTOR_ANALYTICS: false
     ENABLE_LTI_PROVIDER: false
-    ENABLE_MKTG_SITE: false
+    ENABLE_MKTG_SITE: true
```

Comment:
Are there any docs on what this setting does?

Reply:
Ok. I see that it's required to enable https://edx.readthedocs.io/projects/edx-platform-technical/en/latest/featuretoggles.html#featuretoggle-FEATURES['ENABLE_COURSE_HOME_REDIRECT']

I just want to be careful about side effects. What do we need to test to make sure we haven't broken anything?

Reply:
Here are the things that we should test:

- Course about page redirect
- LMS Footer links
- Links in Emails generated from edX i.e. instructor emails

Reply:
@pdpinch Should I merge this PR? Does it look good enough to you to be merged?